tests/test_basic_functionality.py

```diff
@@ -555,6 +557,32 @@ def test_get_chained_default(nested_dict: dict[str, Any]) -> None:
         lct.get(["outer_layer_3", "whoops", "missing_key"], "foo")
 
 
+def test_get_defaults_and_layers() -> None:
+    lct = LayeredConfigTree(layers=["base", "override"])
+    lct.update(
+        {
+            "outer": {
+                "inner": "base-value",
+            }
+        },
+        layer="base",
+    )
+    lct.update(
+        {
+            "outer": {
+                "new-inner": "new-value",
+            }
+        },
+        layer="override",
+    )
+    assert (
+        lct.get(["outer", "new-inner"])
+        == lct.get(["outer", "new-inner"], layer="override")
+        == "new-value"
+    )
+    assert lct.get(["outer", "new-inner"], "missing", layer="base") == "missing"
```

Comment:
From Jim: rename "missing" to something more obvious, e.g. "this-value-is-missing-at-this-later"